src/main/resources/META-INF/plugin.xml

```diff
@@ -14,34 +14,75 @@
-        <li>A <a href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/UserGuide.md#language-servers-preferences" >Language Servers preferences page</a> to configure the LSP trace level and the debug port to use to debug the language server.</li>
+        <li>by manually adding <a
+                href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/UserDefinedLanguageServer.md">language
+            server definitions</a>, supporting custom server settings. This approach doesn't require developing a
+            specific IntelliJ plugin.
+        </li>
     </ul>
 
-    <p>You can start with :
-      <ul>
-        <li>
-          <a href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/DeveloperGuide.md" > which explains step by step how to integrate a language server in LSP4J in an external IntelliJ plugin.
-        </li>
-        <li>
-          <a href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/UserGuide.md" > which explains how to use LSP console and Language server preferences.
+    <p>LSP4IJ also provides:</p>
+    <ul>
+        <li>An <a href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/UserGuide.md#lsp-console">LSP Consoles
+            view
+        </a> to track LSP requests, responses, and notifications in a console.
         </li>
-        <li>
-          <a href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/LSPSupport.md" > which explains the LSP support.
+        <li>A <a
+                href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/UserGuide.md#language-servers-preferences">
+            Language Servers preferences page
+        </a> to configure the LSP trace level and the debug port to use to debug the language server.
         </li>
-      </ul>
+    </ul>
+
+    <p>You can start with :
+        <ul>
+            <li>
+                <a href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/DeveloperGuide.md">Developer Guide
+                </a>
+                which explains step by step how to integrate a language server in LSP4J in an external IntelliJ plugin.
+            </li>
+            <li>
+                <a href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/UserDefinedLanguageServer.md">User-defined
+                    language server
+                </a>
+                which explains how to integrate a language server in LSP4J with few settings.
+            </li>
+            <li>
+                <a href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/UserGuide.md">User Guide</a>
+                which explains how to use LSP console and Language server preferences.
+            </li>
+            <li>
+                <a href="https://github.com/redhat-developer/lsp4ij/blob/main/docs/LSPSupport.md">LSP support</a>
```

Comment:
the LSP Support overview, describing which LSP features are implemented, and how.